### PR TITLE
Fishing map/presence choice

### PR DESCRIPTION
**Pull request by j8seangel**
Using <Choice /> component to toggle between fishing and presence layers with new dataviews categories: `fishing` and `presence`

https://user-images.githubusercontent.com/10500650/117045687-1480cf00-ad10-11eb-8b63-4e15d6fd144c.mp4

Also disables presence layer interaction on vessel click.

Blocked by https://github.com/GlobalFishingWatch/api-datasets/pull/25 to remove dataseis mock before merging

Pending:
- [ ] Review performance on toggle as rendering takes some time
- [ ] Enable again the bivariate mode with new UI

### CODE REVIEW

File: applications/fishing-map/public/locales/en/translations.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "common": {
     "active_after": "Active after",
     "active_before": "Active Before",
-    "activity": "Vessel activity",
+    "activity": "Activity",
```
Comment (nerik) [inline applications/fishing-map/public/locales/en/translations.json:23]: ⚠️ Does not work ⚠️

File: applications/fishing-map/src/features/dataviews/dataviews.selectors.ts [hunk 2]
```diff
@@ -106,6 +107,16 @@ export const selectDataviewInstancesResolved = createSelector(
   }
 )
 
+export const selectDataviewInstancesResolved = createSelector(
```
Comment (nerik) [inline applications/fishing-map/src/features/dataviews/dataviews.selectors.ts:110]: Maybe
```suggestion
export const selectDataviewInstancesByActivityCategory = createSelector(
```

**Review by nerik — COMMENTED** [review]
Looks good! Careful with length of labels in other languages (set line height at the minimum)

File: applications/fishing-map/public/locales/en/translations.json (hunk 1 above)
Comment (j8seangel) [reply applications/fishing-map/public/locales/en/translations.json:23]: Thanks! updated in crowdin

**Review by j8seangel — COMMENTED** [review]
That selector is used everywhere and that is why I preserved the name to avoid a lot of changes.

**Pull request merged**